types/twig/twig-tests.ts

```diff
@@ -38,7 +38,7 @@ const compOpts: twig.CompileOptions = {
     }
 };
 
-twig.extendFilter(str, (left: any, ...params: any[]) => {
+twig.extendFilter(str, (left: any, params: any[] | false) => {
```

Comment:
According to the first comment, you should add here a new line trying to call the function but with the `params` argument

Reply:
As I said in my reply to the first comment, that's not true - but I will add a test matching the one in the docs to ensure omitting the second argument doesn't cause an error :)